test/resources/firebase_config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "databaseURL": "https://hipster-chat.firebaseio.com",
+  "projectId": "hipster-chat",
```

Comment:
Please use some other obviously fake project ID. Same for other parameters in these files. See `mock.key.json` for an example.